pilot_urjc_bringup/launch/turtlebot3_sim/nav2_turtlebot3_launch.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2018 Intel Corporation
```

Comment:
Why copy/paste the original launcher in nav2 instead of including it?

Reply:
I just did the same as that was there for the other two launchers [nav2_tiago_launch.py}(https://github.com/MROS-RobMoSys-ITP/Pilot-URJC/blob/master/pilot_urjc_bringup/launch/tiago/nav2_tiago_launch.py) and [nav2_turtlebot2_launch.py](https://github.com/MROS-RobMoSys-ITP/Pilot-URJC/blob/master/pilot_urjc_bringup/launch/turtlebot2_sim/nav2_turtlebot2_launch.py)

All of them have some differences from the original launcher in nav2, they launch the system modes and actually call a `nav2_bringup_launch` that's inside the `pilot_urjc_bringup`